docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+experiments.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration import RoughXSXAmplitudeCal
+    
+    # rabi experiments for qubit 0
+    rabi0 = RoughXSXAmplitudeCal(0, cals, backend=backend, amplitudes=np.linspace(-0.2, 0.2, 27))
+    
+    # rabi experiments for qubit 1
+    rabi1 = RoughXSXAmplitudeCal(1, cals, backend=backend, amplitudes=np.linspace(-0.2, 0.2, 27))
+
+Run the Rabi experiments.
+
+.. jupyter-execute::
+
+    %%time
+    rabi0_data = rabi0.run().block_for_results()
+    rabi1_data = rabi1.run().block_for_results()
+
+Plot the results.
+
+.. jupyter-execute::
+
+    rabi0_data.figure(0)
+
+.. jupyter-execute::
+
+    rabi1_data.figure(0)
+
+Observe the updated parameters for qubit 0.
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[0, ()], parameters="amp"))
+
+5.4 Rough Drag parameter calibration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run rough Drag parameter calibration for the ``X`` and ``SX`` gates. This follows the same procedure
+as above.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration import RoughDragCal
+    
+    cal_drag0 = RoughDragCal(0, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    cal_drag1 = RoughDragCal(1, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    
+    cal_drag0.set_experiment_options(reps=[3, 5, 7])
+    cal_drag1.set_experiment_options(reps=[3, 5, 7])
+    
+    cal_drag0.circuits()[5].draw(output="mpl")
+
+.. jupyter-execute::
+
+    %%time
+    drag0_data = cal_drag0.run().block_for_results()
+    drag1_data = cal_drag1.run().block_for_results()
+
+.. jupyter-execute::
+
+    drag0_data.figure(0)
+
+
+.. jupyter-execute::
+
+    drag1_data.figure(0)
+
+The updated calibrations object:
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[0, ()], parameters="amp"))
+
+6. Simulating 2 qubit interaction characterization via the |CRHamitonian| experiment
+------------------------------------------------------------------------------------
+
+Finally, simulate the |CRHamitonian| characterization experiment.
+
+First, we further configure the backend to run this experiment. This requires: 
+
+- Adding the custom gate used in the experiment as a valid instruction in the
+  :class:`~qiskit.transpiler.Target`. This ensures that the validation checks on the submitted jobs
+  recognize the gate as a valid instruction.
+- Defining the control channel map, which is a dictionary mapping control-target qubit index pairs
+  (given as a tuple) to the control channel index used to drive the corresponding cross-resonance
+  interaction. This is required by the experiment to determine which channel to drive for each
+  control-target pair.
+
+.. jupyter-execute::
+
+    # add the gate to the target
+    from qiskit_experiments.library import CrossResonanceHamiltonian
+    backend.target.add_instruction(
```

Comment:
This is really needed? I think the experiment [dynamically adds calibration](https://github.com/Qiskit/qiskit-experiments/blob/0a9da6a2656515b20fcc6338207b41d1ee4274ee/qiskit_experiments/library/characterization/cr_hamiltonian.py#L395) and transpiler and scheduler can schedule it without having defs in the Target. If it doesn't probably it's a bug.

Reply:
If I don't add this an error is raised.